docs/rules/binding-positions.md

```diff
@@ -13,6 +13,7 @@ The following patterns are considered warnings:
 html`<x-foo ${expr}="bar">`;
 html`<x-foo></${expr}>`;
 html`<${expr} attr="bar">`;
+html`<!-- ${expr} -->`;
```

Comment:
Not sure if it's worth mentioning that commented expressions can be suppressed by escaping the `$`:
```js
html`<!-- \${expr} -->`;
```

Reply:
I'm wrong again, sorry for the noise.

Reply:
It does indeed, I've added it to the docs and the tests